### ISSUE TITLE
(mars theme) aria-current="page" should be retained on menu for eg /blog/page/N/

### ISSUE THREAD

**Issue by codemonkeynorth** [frontity/frontity#846]
(sorry if this is not the place for this... should these minor issues just be raised on the forum?)

**_Bug report_**

**Expected behavior**

When visiting eg `/blog/page/2/` the "Blog" item should still be highlighted in the menu. 
this is currently created by
`aria-current={state.router.link === link ? "page" : undefined}`
and so only works on the first page

**Observed behavior**

Blog item is underlined in  menu for the first page, but not for subsequent

**Steps involved to reproduce the problem**

* go to https://5hok7.sse.codesandbox.io/blog/
* click "older posts"

**Comment by luisherranz** [comment]
Thanks for the report, @codemonkeynorth.

As this is something that is up to the users and not the Frontity framework (even though it would be probably good to have in our starter theme) I'm going to label it as `low-priority` and `good-first-issue`.

PRs are welcomed 🙂

**Comment by rubenlorenzo** [comment]
Hello @luisherranz, I am looking to resolve the bug. Can you assign the issue to me?

**Comment by luisherranz** [comment]
Done! 🙂

**Comment by rubenlorenzo** [comment]
Hello @luisherranz , I can not pass the linter. The following appears to me:

> Warning: React version not specified in eslint-plugin-react settings. See https://github.com/yannickcr/eslint-plugin-react#configuration .
.../frontity/packages/mars-theme/src/components/nav.js
  4:1  error  Missing JSDoc @returns declaration        jsdoc/require-returns
  9:0  error  Missing JSDoc @param "root0" description  jsdoc/require-param-description

**Comment by luisherranz** [comment]
Open a PR and I'll take a look, but it only seems like there is a TSDoc comment missing.
